packages/dev/docs/pages/blog/drag-and-drop.mdx

```diff
@@ -0,0 +1,244 @@
+      } = e;
+      // Only remove the dragged items if they aren't dropped inside the source list
+      if (dropOperation === 'move' && !isInternal) {
+        list.remove(...keys);
+      }
+    }
+    /*- end highlight -*/
+  });
+
+  return (
+    <ListView
+      aria-label={props['aria-label']}
+      selectionMode="multiple"
+      width="size-3600"
+      height="size-3600"
+      items={list.items}
+      dragAndDropHooks={dragAndDropHooks}>
+      {item => (
+        <Item textValue={item.name}>
+          {item.type === 'folder' ? <Folder /> : <File />}
+          <Text>{item.name}</Text>
+        </Item>
+      )}
+    </ListView>
+  );
+}
+
+function DragBetweenListsExample() {
+  let list1 = useListData({
+    initialItems: [
+      {id: '1', type: 'file', name: 'Adobe Photoshop'},
+      {id: '2', type: 'file', name: 'Adobe XD'},
+      {id: '3', type: 'folder', name: 'Documents'},
+      {id: '4', type: 'file', name: 'Adobe InDesign'},
+      {id: '5', type: 'folder', name: 'Utilities'},
+      {id: '6', type: 'file', name: 'Adobe AfterEffects'}
+    ]
+  });
+
+  let list2 = useListData({
+    initialItems: [
+      {id: '7', type: 'folder', name: 'Pictures'},
+      {id: '8', type: 'file', name: 'Adobe Fresco'},
+      {id: '9', type: 'folder', name: 'Apps'},
+      {id: '10', type: 'file', name: 'Adobe Illustrator'},
+      {id: '11', type: 'file', name: 'Adobe Lightroom'},
+      {id: '12', type: 'file', name: 'Adobe Dreamweaver'}
+    ]
+  });
+
+
+  return (
+    <Flex wrap gap="size-300">
+      <BidirectionalDnDListView list={list1} aria-label="First ListView in drag between list example" />
+      <BidirectionalDnDListView list={list2} aria-label="Second ListView in drag between list example" />
+    </Flex>
+  );
+}
+
+<DragBetweenListsExample />
+```
+
+## Challenges
+
+Drag and drop is a complex and flexible interaction pattern, and accessibility support is relatively uncharted territory, so building a complete drag and drop implementation was a huge challenge.
+
+The HTML drag and drop API is notoriously [quirky](https://www.quirksmode.org/blog/archives/2009/09/the_html5_drag.html), [difficult to use](https://horstmann.com/unblog/2018-12-16/index.html), and [lacking many features](http://tolmasky.com/2009/08/16/on-html-5-drag-and-drop/), and browser implementations often have differences and bugs. It was originally designed in the Internet Explorer 5 days (way back in 1999!), and after being standardized as part of HTML5 in 2009, has been relatively unchanged ever since. That said, it’s the only way to support integration with the operating system, which is a requirement for features such as file uploading, inter-app drag and drop, and dragging between windows, iframes, and JS frameworks. We deal with at least 13 different browser bugs in our implementation, normalizing the behavior for applications that rely on our hooks. We also implement several features on top, such as support for multi-item drags.
```

Comment:
Maybe would be worthwhile linking to our bug list/making a separate browser bug list for the DnD stuff? Wondering if people would be interested to see what we encountered

Reply:
yeah I could link to our wiki. people can search for dnd there I guess